tests/test_config_hub.py

```diff
@@ -8,23 +8,28 @@
 from lightning.fabric.plugins import Precision
 
 
+fixed_pairs = [
+    ("litgpt/pretrain.py", "pretrain/debug.yaml"),
+    ("litgpt/pretrain.py", "pretrain/tinyllama.yaml"),
+    ("litgpt/pretrain.py", "pretrain/tinystories.yaml"),
+    ("litgpt/pretrain.py", "https://raw.githubusercontent.com/Lightning-AI/litgpt/wip/config_hub/pretrain/tinystories.yaml"),
+]
+
+model_pairs = []
+models = ["gemma-2b", "llama-2-7b", "tiny-llama"]
+configs = ["full", "lora", "qlora"]
```

Comment:
This looks good for now, but we might not always have all these combinations. 
In the future, we might want to replace this logic here by just reading the `config_hub` directory and listing each file, inferring the script name from the subfolder.

Reply:
That's a good point. I updated it to collect the configs automatically